### PR TITLE
Add custom syntax and parser support

### PR DESCRIPTION
**Pull request by TrySound**
Fix #79 #96

### CODE REVIEW

**Comment by TrySound** [comment]
Do not merge yet

**Comment by TrySound** [comment]
Okay. Can be merged for now.

File: index.js [hunk 1]
```diff
@@ -34,6 +34,17 @@ function AtImport(options) {
...
+    if (result.opts.syntax) {
+      options.parser = result.opts.syntax.parse
+    }
+    if (result.opts.parser) {
+      options.parser = result.opts.parser
+    }
+    if (options.parser.parse) {
+      options.parser = options.parser.parse
+    }
+
```
Comment (MoOx) [inline index.js]: Can you document this (at least in the CHANGELOG?)

Comment (TrySound) [inline index.js]: It's postcss code part. Everything is in postcss doc. I'll add changelog

**Comment by MoOx** [comment]
Can you rebase?

**Comment by TrySound** [comment]
@MoOx Done

**Pull request merged**